### PR TITLE
Feature: Metrics Update

### PR DESCRIPTION
**Pull request by refugene**
- Remove Constellation related code and renamed Star to App
- Rename `Stats` to `Stat` in the loadOrCreate helper
- Add Arbitrum Sepolia network
- Populate Transaction Entity from RewardFacet

### CODE REVIEW

File: abis/AppFactory.json
Comment (george-openformat) [inline abis/AppFactory.json:1]: only need the "abi" part of this file. lines 2-159

**Review by george-openformat — CHANGES_REQUESTED** [review]
looks good to me! 

just one thing with the AppFactory.json

File: abis/AppFactory.json
Comment (refugene) [reply abis/AppFactory.json:1]: Thanks @george-openformat I have removed the rest of the file . I kept lines 2-259

**Review by george-openformat — APPROVED** [review]

**Review by nup9151f — APPROVED** [review]

**Comment by george-openformat** [comment]
Hey @tinypell3ts we are unable to merge this as it has "code owner review required"

**Review by tinypell3ts — APPROVED** [review]

**Pull request merged**